addons/account_facturx/models/account_move.py

```diff
@@ -142,17 +154,11 @@ def _import_facturx_invoice(self, tree):
         with Form(self.with_context(default_type=default_type)) as invoice_form:
             # Partner (first step to avoid warning 'Warning! You must first select a partner.').
             partner_type = invoice_form.journal_id.type == 'purchase' and 'SellerTradeParty' or 'BuyerTradeParty'
-            elements = tree.xpath('//ram:'+partner_type+'/ram:SpecifiedTaxRegistration/ram:ID', namespaces=tree.nsmap)
-            partner = elements and self.env['res.partner'].search([('vat', '=', elements[0].text)], limit=1)
-            if not partner:
-                elements = tree.xpath('//ram:'+partner_type+'/ram:Name', namespaces=tree.nsmap)
-                partner_name = elements and elements[0].text
-                partner = elements and self.env['res.partner'].search([('name', 'ilike', partner_name)], limit=1)
-            if not partner:
-                elements = tree.xpath('//ram:'+partner_type+'//ram:URIID[@schemeID=\'SMTP\']', namespaces=tree.nsmap)
-                partner = elements and self.env['res.partner'].search([('email', '=', elements[0].text)], limit=1)
-            if partner:
-                invoice_form.partner_id = partner
+            invoice_form.partner_id = find_partner(partner_type)
+
+            # Delivery Partner
+            if 'partner_shipping_id' in self._fields:
```

Comment:
I didn't see that in the original PR :face_with_head_bandage:

Reply:
https://github.com/odoo-dev/odoo/blob/59a34b33f4c3aecd06f231d2a4ae0307ab96b772/addons/account_edi_facturx/models/account_edi_format.py#L202

To avoid a bridge module. It's done the same way in the export : https://github.com/odoo-dev/odoo/blob/59a34b33f4c3aecd06f231d2a4ae0307ab96b772/addons/account_edi_facturx/data/facturx_templates.xml#L168

Obviously it's not ideal, but we discussed it with @smetl and we want to avoid creating a bridge module for such a small thing, especially in stable.